src/zcash/zcashTypes.ts

```diff
@@ -82,13 +82,8 @@ export type ZcashBalances = Omit<
   'totalZatoshi'
 >
 
-export const asZecPublicKey = asObject({
-  birthdayHeight: asNumber,
```

Comment:
The reason we included the birthday height in the public keys is so we could potentially, in the future, have watch-only Zcash wallets. The SDK's don't presently support this use-case, but perhaps they could in the future. I guess we could put back the birthday height if / when that happens.

Otherwise, we could leave the birthday height here, but put an `asOptional` around it so we don't blow up. Another option is to return the birthday height from `derivePublicKey`, but simply fail to mention it in the cleaner. Let me know which direction we want to go with.